site/documentation/content/deployment/create-kubernetes-cluster.md

```diff
@@ -10,19 +10,31 @@ Hono can be deployed to any Kubernetes cluster running version 1.11 or newer. Th
 
 The [Kubernetes web site](https://kubernetes.io/docs/setup/) provides instructions for setting up a (local) cluster on bare metal and/or virtual infrastructure from scratch or for provisioning a managed Kubernetes cluster from one of the popular cloud providers.
 
-## Local Development
+<a name="Local Development"></a>
+## Setting up a local Development Environment
 
 The easiest option is to set up a single-node cluster running on a local VM using the [Minikube](https://github.com/kubernetes/minikube) project.
 This kind of setup is sufficient for evaluation and development purposes.
 Please refer to Kubernetes' [Minikube setup guide](https://kubernetes.io/docs/setup/minikube/) for instructions on how to set up a cluster locally.
 
-When using Minikube, the `minikube tunnel` command should be run in order to support Hono's services being deployed using the *LoadBalancer* type. Please refer to the [Minikube Networking docs](https://github.com/kubernetes/minikube/blob/master/docs/networking.md#loadbalancer-emulation-minikube-tunnel) for details.
+The recommended settings for a Minikube VM used for running Hono's example setup are as follows:
 
-## Production Environment
+* **cpus**: 2
+* **memory**: 8192 (MB)
+* **kubernetes-version**: v1.13.6
```

Comment:
Instead of recommending a specific version (which could get outdated soon in terms of security fixes), I would rather omit the `kubernetes-version` option and let the default be used. Same for the `vm-driver` option.

Reply:
I intended to set kind of a baseline regarding the minimum version we would be willing to support. If we do not set a particular version here, then Minikube will choose the version for us and we might end up with a version that we haven't test yet (too new) or we know does not work (too old?).
WDYT?

Reply:
Well, I wouldn't expect much issues because of a "too new" version, actually.
But, of course we can't be sure. So, ok, with specifying the kubernetes-version we are on the safe side. 
I still would remove the `--vm-driver` option though, letting the default be used.